crates/wasmparser/src/validator/names.rs

```diff
@@ -260,11 +326,11 @@ pub enum KebabNameKind<'a> {
     },
     /// `wasi:http/types@2.0`
     #[allow(missing_docs)]
-    Id {
+    RegistryId {
         namespace: &'a KebabStr,
         package: &'a KebabStr,
-        interface: &'a KebabStr,
-        version: Option<Version>,
+        interface: Option<&'a KebabStr>,
```

Comment:
To keep things somewhat simpler for now could this be left as `&'a KebabStr` and effectively only `a:b/c` works today?

Reply:
The primary motivator here was that when actually using the new implementation imports, there is no `/` generally.  For example, unlocked imports tend to look something like
```
(import (unlocked-dep "foo:five@{>=2.0.0 <3.0.0}") (instance (;0;) (type 1)))
```
It definitely would be nice to simplify if possible, as I added a lot of checks elsewhere to enable the optionality here, but I'm not sure the best way to go about that while simultaneously supporting this use case.